src/components/NavigationBar.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { Navbar, Nav, Image } from 'react-bootstrap'
 import { Link } from "react-router-dom";
+import githubLogo from '../images/githubLogo.png'
```

Comment:
This is part of a separate pull request, please create a separate pull request for it